js/libs/core.js

```diff
@@ -13,20 +12,20 @@ Fliplet.InteractiveMap = (function() {
     emit: function(eventName, data) {
       eventHub.$emit(eventName, data);
     },
-    component(componentName, component) {
```

Comment:
Uhm, why is this not accepted anymore?

Reply:
It is accepted, but I changed it to be consistent with the other functions defined above.